### ISSUE TITLE
Solaris 11.3 and "Failure... No filesystem access enabled"

### ISSUE THREAD

**Issue by noloader** [randombit/botan#852]
```
$ LD_LIBRARY_PATH=. ./botan-test
Testing Botan 2.0.1 (unreleased, revision git:885c96251e931909f783c1bc3fefd9f7663e47d9, distribution unspecified)
Starting tests rng:HMAC_DRBG with seed '149CEC31CF718EA1'
block ran 1 tests 1 FAILED
Failure 1: No filesystem access enabled.
stream ran 1 tests 1 FAILED
Failure 1: No filesystem access enabled.
hash ran 1 tests 1 FAILED
Failure 1: No filesystem access enabled.
mac ran 1 tests 1 FAILED
Failure 1: No filesystem access enabled.
modes ran 1 tests 1 FAILED
Failure 1: No filesystem access enabled.
pbkdf ran 1 tests 1 FAILED
Failure 1: No filesystem access enabled.
HMAC_DRBG(SHA-1) ran 240 tests in 39.92 msec all ok
HMAC_DRBG(SHA-224) ran 240 tests in 101.87 msec all ok
HMAC_DRBG(SHA-256) ran 240 tests in 73.25 msec all ok
HMAC_DRBG(SHA-384) ran 240 tests in 182.11 msec all ok
HMAC_DRBG(SHA-512) ran 240 tests in 186.53 msec all ok
HMAC_DRBG(SHA-512-256) ran 240 tests in 141.87 msec all ok
...
```

---

```
solaris:botan$ uname -a
SunOS solaris 5.11 11.3 i86pc i386 i86pc
solaris:botan$ uname -m
i86pc
solaris:botan$ uname -s
SunOS
```

**Comment by noloader** [comment]
It looks like the change for [Issue 824](https://github.com/randombit/botan/issues/824) applies here as well:

```
$ git diff  src/build-data/os/solaris.txt
diff --git a/src/build-data/os/solaris.txt b/src/build-data/os/solaris.txt
index 49962e8..38bd5d4 100644
--- a/src/build-data/os/solaris.txt
+++ b/src/build-data/os/solaris.txt
@@ -8,6 +8,7 @@ install_cmd_exec '/usr/ucb/install -m 755'
 <target_features>
 posix_mlock
 gettimeofday
+readdir
 threads
 filesystem
 </target_features>
```

**Comment by randombit** [comment]
Fixed in 9964d632f, thanks.
